angularjs-portal-home/src/main/webapp/my-app/marketplace/controllers.js

```diff
@@ -2,13 +2,15 @@
 
 define(['angular', 'jquery', 'require'], function(angular, $, require) {
 
-    var app = angular.module('my-app.marketplace.controllers', []);
+  var app = angular.module('my-app.marketplace.controllers', []);
 
-    app.controller('marketplaceCommonFunctions',
-      ['googleCustomSearchService', 'miscSearchService', 'layoutService', 'marketplaceService', 'miscService', 'MISC_URLS', '$sessionStorage',
-       '$localStorage','$rootScope', '$scope', '$routeParams', '$timeout', '$location',
-       function(googleCustomSearchService, miscSearchService, layoutService, marketplaceService, miscService,MISC_URLS, $sessionStorage,
-        $localStorage, $rootScope, $scope, $routeParams, $timeout, $location){
+  app.controller('marketplaceCommonFunctions', ['googleCustomSearchService', 'miscSearchService', 'layoutService', 'marketplaceService',
+    'miscService', 'MISC_URLS', '$sessionStorage', '$localStorage','$rootScope', '$scope', '$routeParams', '$timeout', '$location',
+    function(googleCustomSearchService, miscSearchService, layoutService, marketplaceService, miscService, MISC_URLS, $sessionStorage,
+             $localStorage, $rootScope, $scope, $routeParams, $timeout, $location) {
+
+      var currentThemePrimary = $sessionStorage.portal.theme.materialTheme.primary['500'];
+      $scope.primaryColorRgb = 'rgb('+ currentThemePrimary.value[0] + ',' + currentThemePrimary.value[1] + ',' + currentThemePrimary.value[2] + ')';
```

Comment:
This could create issues if we didn't create a custom palette for the current theme. One may suggest using `$mdColors` instead: https://material.angularjs.org/latest/api/service/$mdColors